src/main/java/com/money_account_service/utility/UserServiceClient.java

```diff
@@ -0,0 +1,44 @@
+package com.money_account_service.utility;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.money_account_service.dtos.response.AuthorizeResponseDto;
+import lombok.AllArgsConstructor;
+import okhttp3.Call;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.io.IOException;
+
+@AllArgsConstructor
+public class UserServiceClient {
+
+    private OkHttpClient okHttpClient;
+    public static final String AUTHORIZE_URL = "https://localhost:8081/authorize";
```

Comment:
1. It feels like we should have a configuration for such a thing, rather than a static field in code. Different environments can have different addresses.
2. It looks like we would advantage from adding a bit more to the path. Example pattern we could use `/service/api/v1/authorize`. This would help with routing requests (if we had a load balanacer) and versioning the API.